### PR TITLE
Add PostCSS syntaxes support

### PR DESCRIPTION
**Pull request by ai**
Hi. PostCSS is a very popular tool and many developers use it for preprocessing as well. For example, WebStorm already supports PostCSS.

I added two popular file ext for PostCSS preprocessors: `.pcss` (just CSS syntax with nesting/mixins) and `.sss` (Sass-like indent-based syntax).

### CODE REVIEW

**Comment by ai** [comment]
Oops, seems like I need force `Syntax-highlighting-for-PostCSS` to got `LICENSE` file

**Comment by ai** [comment]
I added issue to grammar repo: https://github.com/hudochenkov/Syntax-highlighting-for-PostCSS/issues/19

**Comment by ai** [comment]
Travis CI is broken because of license check. But `Syntax-highlighting-for-PostCSS` already added LICENSE. Maybe we need to clean cache on Travis CI?

File: vendor/licenses/grammar/Syntax-highlighting-for-PostCSS.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+type: grammar
+name: Syntax-highlighting-for-PostCSS
+license: other
```
Comment (hudochenkov) [inline vendor/licenses/grammar/Syntax-highlighting-for-PostCSS.txt]: Try `license: mit` here. Maybe it'll fix CI fail.

And maybe you should copy license text as well. Other files in this directory have license text in them.

**Comment by ai** [comment]
Hm. I will try to clean make cache and make `add-grammar` again

**Comment by ai** [comment]
@kivikakk I fixed tests =^_^=

**Pull request merged**